frontend/src/components/common/ReleaseNotes/UpdatePopup.tsx

```diff
@@ -2,11 +2,52 @@ import { Button, Snackbar } from '@material-ui/core';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-function UpdatePopup(props: { releaseDownloadURL: string }) {
+function UpdatePopup(props: {
+  releaseDownloadURL?: string | null;
+  fetchingRelease?: boolean;
+  releaseFetchFailed?: boolean;
+}) {
   const [show, setShow] = React.useState(true);
-  const { releaseDownloadURL } = props;
+  const [releaseFetchShow, setReleaseFetchShow] = React.useState(true);
+  const { releaseDownloadURL, fetchingRelease, releaseFetchFailed } = props;
   const { t } = useTranslation('frequent');
 
+  if (fetchingRelease) {
+    return (
+      <Snackbar
```

Comment:
This and the next Snackbar declared below do not show up in the same location as the current release related snackbar. They should be consistent.

Reply:
fixed